Source/Scene/Camera.js

```diff
@@ -2552,6 +2552,23 @@ define([
             destination = this.getRectangleCameraCoordinates(destination, scratchFlyToDestination);
         }
 
+        var sscc = this._scene.screenSpaceCameraController;
+        var ellipsoid = this._scene.globe.ellipsoid;
```

Comment:
In practice, `this._scene.globe` should not be `undefined`, but it may be for testing or debugging. Use `this._scene.mapProjection.ellipsoid`.